python/lsst/analysis/tools/contexts/_baseContext.py

```diff
@@ -68,10 +68,12 @@ class ContextApplier:
     @overload
     def __get__(
         self, instance: AnalysisAction, klass: type[AnalysisAction] | None = None
-    ) -> Callable[[ContextType], None]: ...
+    ) -> Callable[[ContextType], None]:
+        ...
 
     @overload
-    def __get__(self, instance: None, klass: type[AnalysisAction] | None = None) -> ContextApplier: ...
+    def __get__(self, instance: None, klass: type[AnalysisAction] | None = None) -> ContextApplier:
+        ...
 
```

Comment:
Should these really have been changed?

Reply:
I ran black and isort since I know analysis_tools is picky about linting, and this is what it did 🤷‍♀️